src/util/Util.js

```diff
@@ -44,6 +44,9 @@ define(['okta'], function (Okta) {
       var errorMsg = Okta.loc('errors.' + xhr.responseJSON.errorCode, 'login');
       if (errorMsg.indexOf('L10N_ERROR[') === -1) {
         xhr.responseJSON.errorSummary = errorMsg;
+        if(xhr.responseJSON.errorCauses[0] && xhr.responseJSON.errorCauses[0].errorSummary) {
```

Comment:
Can you add a comment explain how it solve the problem
i.e.: BaseForm will consume the `errorCauses` if it is an array... That takes the priority of the `errorSummary`.... Delete the `errorCauses` array since we want to show the translated message by `errorCode`...

Reply:
yes, please add that comment cause I do not understand how this is fixing an error

Reply:
Talked to Manuel offline, makes sense now to me. 
We should just move ```delete xhr.responseJSON.errorCauses;``` to line 41, in the same if block where we are overriding the top errorSummary.